go/libkb/id_table.go

```diff
@@ -453,6 +453,19 @@ func (l *TrackChainLink) GetTrackedKeys() ([]TrackedKey, error) {
 
 	var res []TrackedKey
 
+	// If the eldest kid in this tracking statement is a PGP key, it's
+	// implicitly tracked, in addition to the "pgp_keys" section that follows.
+	eldestKeyJSON := l.payloadJSON.AtPath("body.track.key")
+	eldestTrackedKey, err := trackedKeyFromJSON(eldestKeyJSON)
+	if err != nil {
```

Comment:
Maybe it's best to just skip this block in the error case, instead of erroring out.